models/models.go

```diff
@@ -147,12 +147,14 @@ type CveDetail struct {
 
 // IsNvd returns true if NVD contents
 func (c CveDetail) IsNvd() bool {
-	return c.NvdJSON != nil && c.NvdJSON.ID != 0
+	return c.NvdJSON != nil && c.NvdJSON.ID != 0 ||
+		c.NvdJSON != nil && c.Jvn == nil //Redis
 }
 
 // IsJvn returns true if JVN contents
 func (c CveDetail) IsJvn() bool {
-	return c.Jvn != nil && c.Jvn.ID != 0
+	return c.Jvn != nil && c.Jvn.ID != 0 ||
+		c.NvdJSON == nil && c.Jvn != nil //Redis
 }
```

Comment:
I don't understand why IsNvd evaluates Jvn to be nil, and why IsJvn evaluates Nvd to be nil as well.